src/menu.ts

```diff
@@ -150,6 +150,16 @@ function getAppMenu( mainWindow: BrowserWindow | null ) {
 				{ role: 'zoomOut' },
 				{ type: 'separator' },
 				{ role: 'togglefullscreen' },
+				{ type: 'separator' },
+				{
+					label: __( 'Float on Top of All Other Windows' ),
```

Comment:
I had the same thought when testing this +1

Reply:
Great Observation,

I've decided to put the menu under View to follow QuickTime settings:

<img width="515" alt="quicktime" src="https://github.com/user-attachments/assets/9f3c327c-8b24-475a-a650-a3bd49af6d32">


I've checked other apps like VLC which displays that menu item under video instead of Window.

<img width="739" alt="vlc" src="https://github.com/user-attachments/assets/e1487a0e-2e1c-47c3-8c37-00299c22a105">

I've also observed that we don't have a "Window" menu item for Windows, we could add it but it would be the only item there.

![windows-menu](https://github.com/user-attachments/assets/18ef61b9-cd35-4959-9eb3-73749901fde7)

Reply:
Good point. To be honest, I don't have a strong preference between the View and Window tabs to place the item so looks good to me to put it under View. My comment was more regarding the discrepancy with the design.